placement-rules-in-sql.md

```diff
@@ -18,6 +18,7 @@ The detailed user scenarios are as follows:
 - Store new data into NVMe storage and store old data into SSDs to lower the cost on data archiving and storage 
 - Schedule the leaders of hotspot data to high-performance TiKV instances
 - Separate cold data to lower-cost storage mediums to improve cost efficiency
+- Support the physical isolation of computing resources between different users, which meets the isolation requirements of different users within an instance, and the isolation requirements of CPU, I/O, memory, and other resources with different mixed loads
```

Comment:
May I ask where is the source? I am not aware of the scenarios.

Reply:
It is translated from https://github.com/pingcap/docs-cn/pull/9279.

Reply:
Then `within an instance` should not be used. Maybe `in a cluster`. We can not do isolation on one machine, only on different machines.

Reply:
```suggestion
- Support the physical isolation of computing resources between different users, which meets the isolation requirements of different users in a cluster, and the isolation requirements of CPU, I/O, memory, and other resources with different mixed loads
```

Reply:
@xhebox addressed